docs/json.md

```diff
@@ -88,9 +88,9 @@ ABAP Type | JSON Schema Type | JSON Schema Additions
 :--- | :---  | :---
 string | string |
 c | string | `"maxLength": <length of character field>`
-i | integer | `"minimum": -2147483648, "maximum": 2147483647`
+i | integer | `"minimum": -2147483648, "maximum": 2147483647`, "multipleOf": <e.g., 0.01 for 2 integer>`
```

Comment:
I think, we don't add `"multipleOf"` automatically to integers. I suggest we just remove this addition.

```suggestion
i | integer | `"minimum": -2147483648, "maximum": 2147483647`
```